ingest-log4j-vulns/scan-for-log4j.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if [ -z "$J1_ACCOUNT" ] || [ -z "$J1_ACCESS_TOKEN" ]; then
+  echo "You must export both J1_ACCOUNT and J1_ACCESS_TOKEN vars!"
+  exit 2
+fi
+
+SCANPATH=${1:-/}
+if [ "$SCANPATH" = "/scan" ]; then
```

Comment:
This is needed to ensure Dockerized runs have access to a persistent, volume-mounted filesystem so scope remains stable across runs.